package-lock.json

```diff
@@ -3596,7 +3596,7 @@
 				},
 				"load-json-file": {
 					"version": "2.0.0",
-					"resolved": "https://registry.npmjs.org/load-json-file/-/load-json-file-2.0.0.tgz",
+					"resolved": "http://registry.npmjs.org/load-json-file/-/load-json-file-2.0.0.tgz",
```

Comment:
Please change back to https.

Reply:
reverted https; no clue where this comes from..